src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+    sonar.projectKey=<copy-this-from-the-sonarqube-server-webpage>
```

Comment:
Can we prompt the user for this while creating the solution?

Reply:
Nope. It can be added only after you add the project to github